pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
```

Comment:
It's important that composer is updated, as it might not work correctly if it isn't.